examples/sudoku/server/sudoku.hpp

```diff
@@ -0,0 +1,216 @@
+//  Copyright (c) 2016 Satyaki Upadhyay
+//
+//  Distributed under the Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#if !defined(HPX_SUDOKU_EXAMPLE_SERVER)
+#define HPX_SUDOKU_EXAMPLE_SERVER
+
+#include <iostream>
+
+#include <hpx/include/async.hpp>
+#include <hpx/include/util.hpp>
+#include <hpx/hpx_fwd.hpp>
+#include <hpx/include/components.hpp>
+#include <hpx/runtime/applier/applier.hpp>
+#include <hpx/runtime/threads/thread_data.hpp>
+#include <hpx/runtime/components/component_type.hpp>
+#include <hpx/runtime/actions/component_action.hpp>
+#include <hpx/runtime/serialization/serialize.hpp>
```

Comment:
Please try to include headers from `hpx/include` instead, if possible. We'd like to slowly move away from exposing the internal headers.